tuning/utils/preprocessing_utils.py

```diff
@@ -29,8 +29,48 @@
 logger = logging.get_logger("sft_trainer_preprocessing")
 
 
+def _is_pretokenized_dataset(dataset: Dataset):
+    return ("input_ids" in dataset.column_names) and ("labels" in dataset.column_names)
+
+
 def validate_data_args(data_args: configs.DataArguments, packing: bool):
 
+    # validation for pretokenized datasets case
```

Comment:
this kid of checks can be hard to parse, i suggest to document in plain english above.

Reply:
@fabianlim documented them whereever possible thanks